libyul/optimiser/FullInliner.cpp

```diff
@@ -363,8 +363,5 @@ Statement BodyCopier::operator()(FunctionDefinition const&)
 
 YulName BodyCopier::translateIdentifier(YulName _name)
 {
-	if (m_variableReplacements.count(_name))
-		return m_variableReplacements.at(_name);
-	else
-		return _name;
+	return util::valueOrDefault(m_variableReplacements, _name, _name);
 }
```

Comment:
It does seem to have some effect, though minimal.

Here's the time taken by `BodyCoper` when compiling Uniswap on 5 runs I tried:
1. 2.000305 s (before)
1. 1.972689 s (before)
1. 1.905709 s (after)
1. 1.910145 s (after)
1. 1.962662 s (after)

So it's consistently lower, but the variance of the results is on the same order as the difference between them so it could very well turn out that will more runs the difference disappears.

For context, the inliner as a whole takes ~3.5 s and the whole compilation process somewhere around 110-120 s.

Reply:
Measuring `BodyCopier::translateIdentifier()` itself, performance concerns seem to be negligible. 2250827 calls take takes 0.008033 s in total, compared to ~120 s running time of full compilation.